docs/configuration.rst

```diff
@@ -149,6 +149,14 @@ Toolbar options
   the request doesn't originate from the toolbar itself, EG that
   ``is_toolbar_request`` is false for a given request.
 
+* ``TOOLBAR_LANGUAGE``
+
+  Default: ``None``
+
+  The language that will be used to render the toolbar. If no value is supplied
+  then the current language will be used. After the toolbar is rendered, the
+  language is set back to the previous one. A common case is to set this to
+  ``en-us`` since most developers use english on projects.
```

Comment:
This is pretty good. I think it can be improved if we explain the scenario that would be useful though. What do you think?

> The language used to render the toolbar. If no value is supplied, then the application's current language will be used. This setting can be used to render the toolbar in a different language than what the application is rendered in. For example, if you wish to use English for development, but want to render your application in French, you would set this to ` ``"en-us"`` ` and ` (``LANGUAGE_CODE``)[https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-LANGUAGE_CODE] ` to `fr`.

Reply:
That's great!
English is not my primary language, so the text might be have a lot of spelling problems :laughing:

Reply:
I had pushed a new commit with this change.